### PR TITLE
fix visdiff animation

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers using a static image for that one gif

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 86135b2fa02e...b5ff99a22aaf introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Menubar-Private_Async_Writing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486421419274-86135b2fa02e-b5ff99a22aaf-linux/Menubar-Private_Async_Writing-86135b2fa02e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486421419274-86135b2fa02e-b5ff99a22aaf-linux/Menubar-Private_Async_Writing-b5ff99a22aaf.png)  
   Menubar-Private_Async_Writing comparison
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486421419274-86135b2fa02e-b5ff99a22aaf-linux/Icon-Icon_Image-86135b2fa02e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486421419274-86135b2fa02e-b5ff99a22aaf-linux/Icon-Icon_Image-b5ff99a22aaf.png)  
   Icon-Icon_Image comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 86135b2fa02e...b5ff99a22aaf introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Menubar-Private_Async_Writing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486485159281-86135b2fa02e-b5ff99a22aaf-linux/Menubar-Private_Async_Writing-86135b2fa02e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486485159281-86135b2fa02e-b5ff99a22aaf-linux/Menubar-Private_Async_Writing-b5ff99a22aaf.png)  
   Menubar-Private_Async_Writing comparison
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486485159281-86135b2fa02e-b5ff99a22aaf-linux/Icon-Icon_Image-86135b2fa02e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486485159281-86135b2fa02e-b5ff99a22aaf-linux/Icon-Icon_Image-b5ff99a22aaf.png)  
   Icon-Icon_Image comparison
</summary>

File: shared/common-adapters/icon.constants.js [hunk 1]
```diff
@@ -750,6 +750,11 @@ const iconMeta_ = {
     extension: 'gif',
     require: require('../images/icons/icon-loader-uploading-16.gif'),
   },
+  'icon-loader-uploading-static-16': {
+    isFont: false,
+    extension: 'gif',
+    require: require('../images/icons/icon-loader-uploading-static-16.gif'),
```
Comment (chromakode) [inline shared/common-adapters/icon.constants.js]: I'm confused by this. Isn't there an `'icon-loader-uploading-16-static'` above? And why are the extensions of this one ".gif"?

Comment (chrisnojima) [inline shared/common-adapters/icon.constants.js]: guess that's a weird half-attempt at doing this. the code for it never existed and it turns out those actually are animated. i deleted them

**Comment by keybase-ci-visdiff** [comment]
The commits 86135b2fa02e...b0f3da6bea64 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Menubar-Private_Async_Writing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486495306864-86135b2fa02e-b0f3da6bea64-linux/Menubar-Private_Async_Writing-86135b2fa02e.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1486495306864-86135b2fa02e-b0f3da6bea64-linux/Menubar-Private_Async_Writing-b0f3da6bea64.png)  
   Menubar-Private_Async_Writing comparison
</summary>

**Pull request merged**